clients/drcachesim/tools/filter/type_filter.h

```diff
@@ -90,7 +90,7 @@ class type_filter_t : public record_filter_t::record_filter_func_t {
     {
         per_shard_t *per_shard = reinterpret_cast<per_shard_t *>(shard_data);
         if (entry.type == TRACE_TYPE_MARKER && entry.size == TRACE_MARKER_TYPE_FILETYPE) {
-            if (TESTANY(entry.addr, OFFLINE_FILE_TYPE_ENCODINGS) &&
+            if (TESTANY(OFFLINE_FILE_TYPE_ENCODINGS, entry.addr) &&
```

Comment:
Did not observe any problem: just found by inspection.  Separated into https://github.com/DynamoRIO/dynamorio/pull/6653.

Reply:
Because the flag constant being checked is a single bit I think correctness is assured.

Reply:
I didn't mean we had to separate it into a separate PR. Seems a quick fix we can include here. Anyway, either way is fine.

> Because the flag constant being checked is a single bit I think correctness is assured.

I think with TESTANY the order doesn't matter, no matter the bit count. For TESTALL, incorrect param order would've been a correctness bug.